doc/source/reference/index.rst

```diff
@@ -15,5 +15,6 @@ code.
    api/api
    configuration
    python_introduction
+   demeshening
```

Comment:
I'm also fixing this in #3368. It's likely that fixing it twice will result in a duplicate entry instead of a single one, can you drop it here ?

Reply:
Sure